contrib/terraform/openstack/kubespray.tf

```diff
@@ -88,6 +89,7 @@ module "compute" {
   extra_sec_groups                             = var.extra_sec_groups
   extra_sec_groups_name                        = var.extra_sec_groups_name
   group_vars_path                              = var.group_vars_path
+  port_security_enabled                                = var.port_security_enabled
```

Comment:
Here needs to be updated for passing tf-validate-openstack.

Reply:
Fixed, thanks for the hint!